pmmdemo/provision_scripts/percona_server_80.yml

```diff
@@ -123,6 +125,12 @@ write_files:
       #max_slowlog_size=3G
       #max_slowlog_files=3
 
+      # InnoDB
+      #
+      innodb_buffer_pool_size=1G
+      innodb_flush_method=O_DIRECT
+      innodb_adaptive_hash_index=OFF
```

Comment:
Disabling the variable also turns off two graphs in the InnoDB Details dashboard. I'd leave AHI enabled :taco: